skiko/src/commonMain/kotlin/org/jetbrains/skia/ImageFilter.kt

```diff
@@ -381,6 +381,46 @@ class ImageFilter internal constructor(ptr: NativePointer) : RefCnt(ptr) {
             }
         }
 
+        fun makeRuntimeShader(runtimeShaderBuilder: RuntimeShaderBuilder, shaderName: String, input: ImageFilter?): ImageFilter {
+            return try {
+                Stats.onNativeCall()
+                interopScope {
+                    ImageFilter(
+                        _nMakeRuntimeShader(
+                            runtimeShaderBuilderPtr = getPtr(runtimeShaderBuilder),
+                            childShaderName = shaderName,
+                            input = getPtr(input)
+                        )
+                    )
+                }
+            } finally {
+                reachabilityBarrier(runtimeShaderBuilder)
+                reachabilityBarrier(input)
+            }
+        }
+
+
+        fun makeRuntimeShader(runtimeShaderBuilder: RuntimeShaderBuilder, shaderNames: Array<String?>, inputs: Array<ImageFilter?>): ImageFilter {
+            return try {
```

Comment:
It will be safer, if we add a size check:
```
require(shaderNames.size == inputs.size)
```

Reply:
Added